crates/biome_dependency_graph/src/snapshots/biome_dependency_graph__dependency_graph__tests__resolve_exports.snap

```diff
@@ -0,0 +1,138 @@
+---
+source: crates/biome_dependency_graph/src/dependency_graph.tests.rs
+expression: data.exports
+---
+{
```

Comment:
It seems like we could use own formatter for better snapshots :) 

I want to give it a shot